CMakeLists.txt

```diff
@@ -1,6 +1,6 @@
 # Icinga 2 | (c) 2012 Icinga GmbH | GPLv2+
 
-cmake_minimum_required(VERSION 2.8.8)
+cmake_minimum_required(VERSION 3.5.2)
```

Comment:
Why did you choose this specific version, i.e. is there a better reason than "because we can." like in #9706?